iphone/Classes/TiUIAttributedStringProxy.m

```diff
@@ -20,13 +20,21 @@ - (void)_destroy
 
 - (void)_initWithProperties:(NSDictionary *)properties
 {
-  NSString *text = [properties valueForKey:@"text"];
+  NSString *html = [properties valueForKey:@"html"];
+  if (html) {
+     _attributedString= [[NSAttributedString alloc] initWithData:[html dataUsingEncoding:NSUTF8StringEncoding]
+                             options:@{NSDocumentTypeDocumentAttribute: NSHTMLTextDocumentType,
+                                       NSCharacterEncodingDocumentAttribute: @(NSUTF8StringEncoding)} 
+                  documentAttributes:nil error:nil];
+  } else {
+    NSString *text = [properties valueForKey:@"text"];
   if (!text) {
     DebugLog(@"[WARN] Ti.UI.AttributedString.text not set");
   }
   _attributedString = [[NSMutableAttributedString alloc] initWithString:text];
+  
+  }
   attributes = [[NSMutableArray alloc] init];
-  [super _initWithProperties:properties];
```

Comment:
Why would this need to be removed?

Reply:
mmm right, I've assumed that if you set a HTML it's pre-formatted, but you could want to apply other styles. Will address it. Thanks!